File: templates/python/matrix_tensor_state_map.py
"""matrix_tensor_state_map: 矩阵与张量可视化状态热力图（heatmap 模式，合成数据）."""
import sys, os
sys.path.insert(0, os.path.join(os.path.dirname(os.path.abspath(__file__)), '..', '..', '_utils', 'python'))
from generated_patterns import make_template_figure


def make_figure(title='matrix and tensor visualization: state heatmap'):
    return make_template_figure('heatmap', seed=4403, title=title, domain='matrix and tensor visualization', topic='state heatmap')


if __name__ == '__main__':
    fig = make_figure()
    fig.savefig(__file__.replace('.py', '.png'), dpi=150)

File: templates/matlab/matrix_tensor_state_map.m
function fig = matrix_tensor_state_map()
    addpath(fullfile(fileparts(mfilename('fullpath')),'..','..','_utils','matlab'));
    fig = generated_template_figure('heatmap', 4403, 'matrix and tensor visualization: state heatmap', 'matrix and tensor visualization', 'state heatmap');
end
